docs/index.md

```diff
@@ -0,0 +1,140 @@
+# AMD Device Metrics Exporter
+
+AMD Device Metrics Exporter enables Prometheus-format metrics collection for AMD GPUs in HPC and AI environments. It provides detailed telemetry including temperature, utilization, memory usage, and power consumption.
+
+## Features
+
+- Prometheus-compatible metrics endpoint
+- Rich GPU telemetry data
+- Kubernetes integration
+- Slurm integration support
+- Configurable service ports
+- Container-based deployment
+
+## Prerequisites
+
+### System Requirements
+
+- Ubuntu 22.04 or later
+- ROCm 6.2.0
+- Docker (or a Docker-compatible container runtime)
+
+## Installation Options
```

Comment:
Please remove the below installation instructions from the main index.md page and instead include what is stated in the Device Metrics Exporter Guide doc around mentioning the 3 different installation methods:

1.	Install on Kubernetes via the GPU Operator Helm Chart (Bundled method)
2.	Install on Kubernetes with the Metrics Exporter Helm Chart (Standalone)
3.	Run via a Docker Container (Standalone)